notebooks/data-sources/oc-github-repo/github_PR_EDA.ipynb

```diff
@@ -2,17 +2,19 @@
  "cells": [
```

Comment:
<p>missing axes and graph titles for the plot</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/293/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
added :)